pkg/reconciler/revision/table_test.go

```diff
@@ -181,10 +69,16 @@ func TestReconcile(t *testing.T) {
 		// are necessary.
 		Objects: []runtime.Object{
 			rev("foo", "stable-reconcile", WithLogURL, AllUnknownConditions,
-				WithContainerStatuses([]v1.ContainerStatuses{{Name: "stable-reconcile"}})),
+				WithContainerStatuses([]v1.ContainerStatuses{
+					{Name: "stable-reconcile",
+						ImageDigest: "busybox@sha256:deadbeef"},
+				})),
```

Comment:
```suggestion
				WithContainerStatuses([]v1.ContainerStatuses{{
					Name: "stable-reconcile",
					ImageDigest: "busybox@sha256:deadbeef"},
				})),
```

(applies to all occurrences of this)